src/common/state/PickAttributes.C

```diff
@@ -1121,6 +1139,8 @@ PickAttributes::CreateNode(DataNode *parentNode, bool completeSave, bool forceAd
         node->AddNode(new DataNode("removeLabelTwins", removeLabelTwins));
     }
 
+    // previousPoint is not persistent and should not be saved.
+    // showDistanceToPrevious is not persistent and should not be saved.
```

Comment:
Why is showDistanceToPrevious not persistent? I guess I could got either way. @biagas thoughts?

Reply:
Persistent means it gets saved to config files correct? Then I guess I could make a case where a user might want this always to be set, so ability to save settings would be good.

Reply:
I didn't really know what persistent meant. I generally just try to match the variables that are already there, but I'll update this.

Reply:
Done.